code/game/objects/items/devices/flashlight.dm

```diff
@@ -79,18 +79,67 @@
 						to_chat(user, "<span class='notice'>[M]'s pupils narrow.</span>")
 
 			if("mouth")
+
 				if((M.head && M.head.flags_cover & HEADCOVERSMOUTH) || (M.wear_mask && M.wear_mask.flags_cover & MASKCOVERSMOUTH))
 					to_chat(user, "<span class='notice'>You're going to need to remove that [(M.head && M.head.flags_cover & HEADCOVERSMOUTH) ? "helmet" : "mask"] first.</span>")
 					return
 
+				var/their = M.p_their()
+
+				var/list/mouth_organs = new
+				for(var/obj/item/organ/O in M.internal_organs)
+					if(O.zone == "mouth")
+						mouth_organs.Add(O)
+				var/organ_list = ""
+				for(var/I = 1, I <= mouth_organs.len, I++)

```

Comment:
for(var/I in 1 to mouth_organs.len)
also you need to check if(LAZYLEN(mouth_organs)) first so it doesn't fuck up if there are none